browser/speedreader/speedreader_service_factory.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_SPEEDREADER_SPEEDREADER_SERVICE_FACTORY_H_
+#define BRAVE_BROWSER_SPEEDREADER_SPEEDREADER_SERVICE_FACTORY_H_
+
+#include "base/memory/singleton.h"
+#include "components/keyed_service/content/browser_context_keyed_service_factory.h"
+#include "components/keyed_service/core/keyed_service.h"
+
+class Profile;
+
+namespace speedreader {
+
+class SpeedreaderService;
+
+class SpeedreaderServiceFactory : public BrowserContextKeyedServiceFactory {
```

Comment:
I think this service & factory aren't necessary.
If this is only used for get/set related prefs value, I think just using simple apis seems fine. WDYT?

Reply:
Yeah, I agree, though I decided to keep it in case we would add a user preference (and would need to keep a pref registrar), or something like this.

Reply:
Ah, ok. if service will have `registrar` in the future, having service seems reasonable.